src/index.js

```diff
@@ -148,8 +148,9 @@ export default (options = {}) => {
           const map = res.map || null
           if (map) {
             map.file = filepath
+            const dirname = path.dirname(opts.file)
             map.sources = map.sources.map(source =>
-              humanlizePath(path.join(path.dirname(opts.file), source))
+              source.substr(0, dirname.length) === dirname ? source : humanlizePath(path.join(path.dirname(opts.file), source))
```

Comment:
Minor: we should use the defined const `dirname` instead.

Reply:
Good pickup, I have resolved for any other places that can make use of the const also.